docs/api-reference/next/image.md

```diff
@@ -90,29 +90,28 @@ The `<Image />` component optionally accepts the following properties.
 
-When `fixed`, the image dimensions will not change as the viewport changes (no
-responsiveness) similar to the native `img` element.
+| `layout`              | Behavior                                                 | `srcSet` (defaults)                  | `sizes` |
+| --------------------- | -------------------------------------------------------- | ------------------------------------ | ------- |
+| `fill`                | Grow in X and Y axes to fill container                   | `640w`, `750w`, ... `2048w`, `3840w` | `100vw` |
+| `fixed`               | Sized to `width` and `height` exactly                    | `1x`, `2x`                           | N/A     |
+| `intrinsic` (Default) | Scale *down* to fit width of container, up to image size | `1x`, `2x`                           | N/A     |
+| `responsive`          | Scale to fit width of container                          | `640w`, `750w`, ... `2048w`, `3840w` | `100vw` |
 
-When `intrinsic`, the image will scale the dimensions down for smaller viewports
-but maintain the original dimensions for larger viewports.
-
-When `responsive`, the image will scale the dimensions down for smaller
-viewports and scale up for larger viewports.
-Note: the responsive layout may not work correctly if the parent element uses a display value other than `block` such as `display: flex` or `display: grid`.
-
-When `fill`, the image will stretch both width and height to the dimensions of
-the parent element, provided the parent element is relative. This is usually paired with the [`objectFit`](#objectFit) property.
-Ensure the parent element has `position: relative` in their stylesheet.
-
-Try it out:
+> **Note:** `srcSet` is based on [deviceSizes](/docs/basic-features/image-optimization.md#device-sizes) and [imageSizes](/docs/basic-features/image-optimization.md#image-sizes) config.
 
 - [Demo the `fixed` layout](https://image-component.nextjs.gallery/layout-fixed)
+  - When `fixed`, the image dimensions will not change as the viewport changes (no responsiveness) similar to the native `img` element.
 - [Demo the `intrinsic` layout](https://image-component.nextjs.gallery/layout-intrinsic)
+  - When `intrinsic`, the image will scale the dimensions down for smaller viewports, but maintain the original dimensions for larger viewports.
 - [Demo the `responsive` layout](https://image-component.nextjs.gallery/layout-responsive)
+  - When `responsive`, the image will scale the dimensions down for smaller viewports and scale up for larger viewports.
+  - Ensure the parent element uses `display: block` in their stylesheet.
 - [Demo the `fill` layout](https://image-component.nextjs.gallery/layout-fill)
+  - When `fill`, the image will stretch both width and height to the dimensions of the parent element, provided the parent element is relative.
+  - This is usually paired with the [`objectFit`](#objectFit) property.
+  - Ensure the parent element has `position: relative` in their stylesheet.
```

Comment:
The order listed here doesn't match the table. Lets make sure the order matches (I think we should have `intrinsic` first now that I think about it since its default.